lib/run.js

```diff
@@ -50,19 +53,19 @@ function run(config, done) {
                     total: tasks.length,
                     info: info,
                     testCmd: testCmd,
-                    installCmd: "npm i" + (config.registry ? (" --registry " + config.registry) : "")
+                    installCmd: (config.useYarn ? "yarn add" : "npm i") + (config.registry ? (" --registry " + config.registry) : "")
```

Comment:
So here we go, it is supported, but not the way npm does it. You have to have a .npmrc in your repo, thats where yarn is getting the registry informations.

I now throw an error when users try to use a custom registry while yarn is used.